invenio_administration/menu/menu.py

```diff
@@ -62,6 +65,20 @@ def register_menu_entries(self, flask_menu_instance, menu_key="admin_navigation"
                     icon=icon,
                 )
 
+    def register_admin_entry(self, current_menu, endpoint):
+        """Register all menu items to a flask menu instance."""
+        current_menu.submenu("settings.administration").register(
+            f"{endpoint}.dashboard",
+            _(
+                "%(icon)s Administration",
+                icon=make_lazy_string(
+                    lambda: f'<i class="{current_theme_icons.cogs}"></i>'
+                ),
+            ),
+            order=6,
+            visible_when=lambda: current_user.roles.count("admin") > 0,
```

Comment:
I would increase the order as we might add new items and we always want it to be the last one.

```suggestion
            order=100,
            visible_when=lambda: current_user.has_role("admin"),
```